{
  "name": "catch-async-wrapper-express",
  "version": "1.2.2",
  "description": "middleware for handling exceptions inside of async express routes and passing them to your express or custom error handlers",
  "main": "./src/index.js",
  "type": "commonjs",
  "scripts": {
    "start": "node ./src",
    "test": "npx nodemon -q ./tests/index.js"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/samislam/catch-async-wrapper-express"
  },
  "keywords": [
    "catchasync",
    "express-async-handler",
    "express-catch-async",
    "express-async-middleware",
    "express",
    "middleware",
    "trycatch-wrapper",
    "trycatchwrapper",
    "express-try-catch",
    "express-trycatch"
  ],
  "author": "Islam Yamor",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/samislam/catchasync/issues"
  },
  "homepage": "https://github.com/samislam/catchasync#readme",
  "dependencies": {
    "@samislam/checktypes": "^1.0.1",
    "@samislam/log": "^1.1.4"
  }
}
